windows/src/engine/keyman32/hookutils.cpp

```diff
@@ -15,45 +15,44 @@ typedef BOOL
     IN CONST PMINIDUMP_CALLBACK_INFORMATION CallbackParam OPTIONAL
     );
 
-
-DWORD ExceptionMessage(LPSTR Proc, LPEXCEPTION_POINTERS ep)
-{
+DWORD ExceptionMessage(LPSTR Proc, LPEXCEPTION_POINTERS ep) {
   MINIDUMP_EXCEPTION_INFORMATION mei;
   char filename[MAX_PATH], temppath[MAX_PATH];
-  if(GetTempPath(MAX_PATH, temppath) == 0 ||
-      GetTempFileName(temppath, "kmc", 0, filename) == 0)
-    SendDebugMessageFormat(0, sdmGlobal, 0, "Minidump failed to generate temp file name");
-  else
-  {
+  if (GetTempPath(MAX_PATH, temppath) == 0 ||
+      GetTempFileName(temppath, "kmc", 0, filename) == 0) {
+    OutputThreadDebugString("Minidump failed to generate temp file name\n");
+  } 
+  else {
```

Comment:
Not same line?  I know, style nitpick, but still.

Reply:
Thank Visual Studio for this -- that's its default. For now, I am comfortable sticking with this; I would like to do a tabs->spaces cleanup at some point and some of this formatting kinda belongs with that.